utils/azure_users.py

```diff
@@ -133,9 +133,10 @@ def users(self) -> List[AzureUser]:
                 )[1]
                 endpoint = endpoint + skip_token_attribute + request_token
                 
-        for i in range(len(users)):
-            if users[i]['mail'] is None:
-                valid_users.append(users[i])
+        for user in users:
+            user_emails = user['otherMails']
+            if(len(user_emails) != 0 and user_emails[0].split('@')[1] == 'ioet.com'):
```

Comment:
Maybe we could put `user_emails[0].split('@')[1]` into a variable `email_domain`? also why is it only checking for the first one?

Reply:
Because Azure returns that as a list and always will be just one email, and sure we can change that into a variable!